devnotes/documentation.md

```diff
@@ -136,17 +149,29 @@ In order to build coreboot image, follow the steps below:
 
    Answer yes to the prompt and wait for the process to finish.
 
-4. Log into the BMC GUI again at https://<IP>/. Enter the Server power
-   operations (https://<IP>/#/server-control/power-operations) and invoke warm
+4. Log into the BMC GUI again at https://\<BMC_IP\>/. Enter the Server power
+   operations (https://\<BMC_IP\>/#/server-control/power-operations) and invoke warm
```

Comment:
```suggestion
   operations (https://\<BMC_IP\>/#/server-control/power-operations) and invoke cold
```

Cold reboot will be faster (and safer), at least until we have proper OCC support

Reply:
I have changed it to `cold` reboot